packages/paste-core/components/table/stories/index.stories.tsx

```diff
@@ -1006,3 +1007,74 @@ export const Truncation = (): React.ReactNode => {
     </Table>
   );
 };
+
+export const CustomTable = (): React.ReactNode => (
+  <CustomizationProvider
```

Comment:
The only thing you might want to consider here, which is something I've started doing, is grabbing the active theme  from context and providing it to the CustomizationProvider.

This way when the story is in side by side for VRT (one side default, one side dark), the theme being rendered is dynamic and matches the side by side.

```
const activeTheme = useTheme();
<CustomizationProvider
  theme={activeTheme}
  ...
```

Reply:
Ohh yeah good point! Thanks,  I'll make the change right now.